src/glow.ts

```diff
@@ -23,3 +24,52 @@ function _handleTransferToCarbonCreditAuction(event: GlowTransferEvent): void {
   );
   totalGlowBurned.save();
 }
+
+export function handleStake(event: StakeEvent): void {
+  
+  let userAddress = event.params.user.toHexString();
+  let stakeAmount = event.params.amount;
+
+  // TODO: Maybe keep track of the stake amount by user
```

Comment:
should we keep track of the stake amount by user?

Reply:
Are you referencing the total amount staked by a user? We could have this in the subgraph.